packages/TelegramClient-Core.package/TCCUser.class/instance/hash.st

```diff
@@ -0,0 +1,4 @@
+comparing
+hash
+
+	^((self firstName hash bitXor: (self lastName hash)) bitXor: (self id hash)) bitXor: (self username hash)
```

Comment:
You forgot a space after the return ^, but more importantly: could you indent it in a way that makes intent clearer? Took me 10s to figure out that expression 😛 

I'm not sure if this is "correct", but I find it a lot more readable

```
hash

	^ ((self firstName hash
			bitXor: (self lastName hash))
				bitXor: (self id hash))
					bitXor: (self username hash)
```

Reply:
would
```smalltalk
hash

	^ ((self firstName hash 
            bitXor: (self lastName hash)) 
            bitXor: (self id hash)) 
            bitXor: (self username hash)```
work? Indentation looks like this in squeak, GitHub shows tabs differently